File: src/BukkitPE/event/inventory/InventoryCloseEvent.php
<?php
namespace BukkitPE\event\inventory;

use BukkitPE\inventory\Inventory;
use BukkitPE\Player;

class InventoryCloseEvent extends InventoryEvent{
	public static $handlerList = null;

	/** @var Player */
	private $who;

	/**
	 * @param Inventory $inventory
	 * @param Player    $who
	 */
	public function __construct(Inventory $inventory, Player $who){
		$this->who = $who;
		parent::__construct($inventory);
	}

	/**
	 * @return Player
	 */
	public function getPlayer(){
		return $this->who;
	}

}

File: src/main/java/net/BukkitPE/event/inventory/InventoryCloseEvent.java
package net.BukkitPE.event.inventory;

import net.BukkitPE.Player;
import net.BukkitPE.event.HandlerList;
import net.BukkitPE.inventory.Inventory;

/**
 * author: Box
 * BukkitPE Project
 */
public class InventoryCloseEvent extends InventoryEvent {

    private static final HandlerList handlers = new HandlerList();
    private final Player who;

    public InventoryCloseEvent(Inventory inventory, Player who) {
        super(inventory);
        this.who = who;
    }

    public static HandlerList getHandlers() {
        return handlers;
    }

    public Player getPlayer() {
        return this.who;
    }
}
